transport/tchannel/channel_inbound.go

```diff
@@ -102,3 +103,13 @@ func (i *ChannelInbound) Stop() error {
 func (i *ChannelInbound) IsRunning() bool {
 	return i.once.IsRunning()
 }
+
+// Introspect returns the state of the inbound for introspection purposes.
+func (i *ChannelInbound) Introspect() introspection.InboundStatus {
```

Comment:
Rather than returning `string`, I'd eventually prefer to pass in an `io.Writer`.